platform/sun50i/devices.c

```diff
@@ -90,3 +92,11 @@ static struct device r_timer0 = {
 	.irq      = IRQ_R_TIMER0,
 	.irqdev   = &r_intc,
 };
+
+static struct device r_twd = {
+	.name     = "r_twd",
+	.clock    = CCU_GATE(R_CCU_GATE_R_TWD),
+	.clockdev = &r_ccu,
+	.drv      = &sunxi_twd_driver,
+	.regs     = DEV_R_TWD,
+};
```

Comment:
Even though we're not using it at the moment, to describe the hardware accurately, this should have
```
.irq = IRQ_R_TWD
.irqdev = &r_intc
```
(This doesn't take up any more space.)

Reply:
Done.